End-to-End-Tests.md

```diff
@@ -436,7 +440,7 @@ Each individual test within each suite gets its own video. The video of each tes
 
 ![Name of video for test gets printed out above test](https://user-images.githubusercontent.com/52176783/118647333-486cf180-b7f2-11eb-999b-9edbbb89b5a7.png)
 
-Note that no videos will be generated on CircleCI due to memory issues when running the video recorder. Further, videos are disabled on GitHub Actions where the `VIDEO_RECORDING_IS_ENABLED` environment variable is set to `0`.
+Note that while recordings are enabled by default, no videos will be generated on CircleCI due to memory issues when running the video recorder. Further, videos are disabled on GitHub Actions where the `VIDEO_RECORDING_IS_ENABLED` environment variable is set to `0`.
```

Comment:
>> Link this to an explanation for how to enable?
> 
> I clarified the later paragraph that describes when recordings are generated

It seems that it is 0 for everything at the moment, so I'm not sure this counts as "recordings enabled by default".

Reply:
By "default" I meant that recordings will be saved so long as the environment variable is not set to 0. However, that's not actually correct--the environment variable has to be set to 1 for videos to be saved. I've updated the wiki page to reflect that